src/activity_actor.h

```diff
@@ -845,6 +845,41 @@ class stash_activity_actor: public activity_actor
         tripoint placement;
 };
 
+class bookbinder_copy_activity_actor: public activity_actor
+{
+    private:
+        int pages = 0;
+        item_location book_binder;
+        std::string r_id = "";
+        std::string r_name = "";
+
+    public:
+
+        bookbinder_copy_activity_actor(
+            int pages,
+            const item_location &book_binder,
+            std::string r_id = "",
+            std::string r_name = ""
```

Comment:
Unless I'm missing something, these should be const references.
```suggestion
            const std::string &r_id = "",
            const std::string &r_name = ""
```